aws-rds-integration/src/main/java/software/amazon/rds/integration/UpdateHandler.java

```diff
@@ -0,0 +1,50 @@
+package software.amazon.rds.integration;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.rds.common.handler.HandlerConfig;
+import software.amazon.rds.common.handler.Tagging;
+
+import java.util.HashSet;
+
+public class UpdateHandler extends BaseHandlerStd {
+    /** Default constructor w/ default backoff */
+    public UpdateHandler() {}
+
+    /** Default constructor w/ custom config */
+    public UpdateHandler(HandlerConfig config) {
+        super(config);
+    }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger
+    ) {
+        // Currently Integration resource only supports Tags update.
+        final ResourceModel desiredModel = request.getDesiredResourceState();
+
+        final Tagging.TagSet previousTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getPreviousSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getPreviousResourceTags()))
+                .resourceTags(new HashSet<>(Translator.translateTagsToSdk(request.getPreviousResourceState().getTags())))
+                .build();
+
+        final Tagging.TagSet desiredTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getDesiredResourceTags()))
+                .resourceTags(new HashSet<>(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags())))
+                .build();
+
+        return ProgressEvent.progress(desiredModel, callbackContext)
```

Comment:
I find it intriguing that the resource lacks the ability to be modified. Could you please provide further elaboration on the significance of these statuses and the circumstances under which they occur or are triggered (Modifying, Syncing, Needs Attention)?
Reference: https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateIntegration.html

Reply:
For now it cannot be modified. The status definition can be found in the official document [here](https://docs.aws.amazon.com/redshift/latest/mgmt/zero-etl-using.describing.html). Currently, it's an entity that creates the synchronization from Aurora to Redshift, and so there is nothing to change really, as-is.

The API status MODIFYING is currently internal, and it should have not made public. In the future, we may support such a capability.